exasol/toolbox/tools/tbx.py

```diff
@@ -3,12 +3,13 @@
 from exasol.toolbox.tools import (
     security,
     workflow,
+    issue,
 )
 
 CLI = typer.Typer()
 CLI.add_typer(workflow.CLI, name="workflow", help="Manage github workflows")
 CLI.add_typer(security.CLI, name="security", help="Security related helpers")
-
+CLI.add_typer(issue.CLI, name="issue", help="issue templates")
```

Comment:
Update help text to be more of a full sentence like above in the workflow command